aspnet/tutorials/first-mvc-app/start-mvc/sample/src/MvcMovie/project.json

```diff
@@ -31,11 +31,11 @@
       ]
     },
     "Microsoft.AspNetCore.Razor.Tools": {
-      "version": "1.0.0-preview2-21343",
+      "version": "1.0.0-*",
```

Comment:
PR https://github.com/aspnet/Docs/pull/1457 will be built with the final RTM bits and should have all the correct version numbers. As mentioned previously, any good changes here I'll manually copy to my PR